examples/data-sources/powerscale_quota/data-source.tf

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# The SmartQuotas module is an optional quota-management tool that monitors and enforces administrator-defined storage limits.
+
+# Returns a list of PowerScale quotas based on filter block
+data "powerscale_quota" "example_quotas" {
+  filter {
```

Comment:
the quotas not support names ?

Reply:
No. Quota only has a poor readable id. And rest API already supports type/path filtering.